cli/tests/lang-node.bats

```diff
@@ -65,16 +64,17 @@ teardown() {
 }
 
 # bats test_tags=catalog
-@test "flox activate works with yarn" {
-  cp -r "$INPUT_DATA/init/node/common/." .
-  cp -r "$INPUT_DATA/init/node/yarn/." .
+@test "auto init matches yarn version to yarn 1.x" {
+  cp -r "$INPUT_DATA/init/node/yarn_1x/." .
   # Files copied from the store are read-only
   chmod -R +w .
 
-  _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/init/node_yarn.json" \
+  _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/init/yarn_1x.json" \
     run "$FLOX_BIN" init --auto-setup
   assert_output --partial "'yarn' installed"
   refute_output "nodejs"
+  run "$FLOX_BIN" list
+  assert_regex "$output" "yarn: yarn \(.+\)"
   run "$FLOX_BIN" activate -- yarn run start
```

Comment:
> We now have a few copies of the `index.js` app and not all of the tests
run an activation with the app but I don't have strong feelings about
duplicating that small amount of content for easier to read tests and
slowing down the other tests by making them all run activations.

note nonblocking: if we're only going to run `yarn run start` for one of the tests, seems like we might want to prefer yarn-berry if that's what most people are using now (idk if that's the case)